client/app/vps/modal/shortcut/reinstall/vps-reinstall.html

```diff
@@ -1,104 +1,89 @@
-                    <label for="language" class="oui-label" data-translate="vps_configuration_reinstall_step1_select_language"></label>
-                    <div class="oui-select">
-                        <select name="language" class="oui-select__input"
-                                data-ng-options="language as $ctrl.getLanguageTraduction(language) for language in $ctrl.template.value.availableLanguage | orderBy: getLanguageTraduction"
-                                data-ng-model="$ctrl.template.language" >
-                            <option value="" data-ng-disabled="true" selected data-translate="vps_configuration_reinstall_step1_language"></option>
-                        </select>
-                        <i class="oui-icon oui-icon-chevron-down" aria-hidden="true"></i>
-                    </div>
-                </div>
+        </oui-field>
 
-                <!-- ssh keys -->
-                <div data-ng-if="$ctrl.template.value && ($ctrl.template.language || $ctrl.template.value.availableLanguage.length === 0) && !$ctrl.isWindows($ctrl.template.value)">
-                    <label for="sshKey" class="oui-label"
-                           data-translate="vps_configuration_reinstall_step1_select_ssh_key"
-                           data-oui-tooltip="{{'vps_configuration_reinstall_step1_select_ssh_key_help' | translate}}"></label>
-                    <div class="oui-select">
-                        <!-- we need style overflow until something is set to handle multiple select-->
-                        <select name="sshkey" id="sshKey" class="oui-select__input"
-                                size="8" multiple
-                                style="overflow-y: scroll"
-                                data-ng-options="sshKey for sshKey in $ctrl.userSshKeys | orderBy"
-                                data-ng-model="$ctrl.template.sshKeys"
-                                data-ng-if="$ctrl.userSshKeys.length > 0">
-                            <option value="" selected></option>
-                        </select>
-                    </div>
-                    <span data-ng-if="!$ctrl.userSshKeys || $ctrl.userSshKeys.count === 0" data-translate="vps_configuration_reinstall_step1_select_ssh_key_no"></span>
-                    <span data-ng-if="$ctrl.template.sshKeys && $ctrl.template.sshKeys.length > 0" data-translate="vps_configuration_reinstall_step1_unselect_ssh_key"></span>
+        <!-- available packages (info box) -->
+        <div data-ng-if="$ctrl.template.packages">
+            <div class="text-center" size="s" data-ng-if="$ctrl.loaders.packages">
+                <oui-spinner></oui-spinner>
+            </div>
+            <div data-ng-if="!$ctrl.loaders.packages">
+                <label class="oui-label" data-translate="vps_configuration_reinstall_step1_packages"></label>
+                <div class="alert alert-info">
```

Comment:
This is an alert then ? `role="alert"`

Reply:
its only used for class names to get similar look and feel. its not an alert. Why it is used in this way, I am not sure. I just retained same code as it is. I have just replaced cui-modal with oui-modal